dist/jowl.spec

```diff
@@ -0,0 +1,88 @@
+Name:           jowl
+Version:        2.0.2
+Release:        1%{?dist}
+Summary:        CLI for JSON operations with Lodash
+
+License:        MIT
+URL:            https://jowl.app
+Source0:        https://github.com/daxelrod/%{name}/archive/v%{version}/%{name}-%{version}.tar.gz
+# Also at       https://registry.npmjs.org/{name}/-/{name}-{version}.tgz
+# however, npmjs.org has a build product which does not contain docs or tests
+
+# The following sources are generated by running
+# nodejs-packaging-bundler against Source0:
+Source1:        %{name}-%{version}-nm-prod.tgz
+Source2:        %{name}-%{version}-nm-dev.tgz
+Source3:        %{name}-%{version}-bundled-licenses.txt
+
+BuildArch:      noarch
+ExclusiveArch:  %{nodejs_arches} noarch
+Requires:       nodejs
+BuildRequires:  nodejs-devel
+BuildRequires:  yarnpkg
+
+%description
+Jowl is a command-line filter for JSON expressions that uses plain JavaScript
+with Lodash. It takes JSON on standard in, and writes pretty-printed JSON to
+standard out.
+
+%prep
+%setup -q -n %{name}-%{version}
+cp %{SOURCE3} .
+# Setup bundled runtime(prod) node modules
+tar xfz %{SOURCE1}
+mkdir -p node_modules
+pushd node_modules
+ln -s ../node_modules_prod/* .
+ln -s ../node_modules_prod/.bin .
+popd
+
+%build
+#nothing to do
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{name}
+cp -pr package.json src/bin/jowl.js src/lib/ \
+    %{buildroot}%{nodejs_sitelib}/%{name}
+# Copy over bundled nodejs modules
+cp -pr node_modules node_modules_prod \
+    %{buildroot}%{nodejs_sitelib}/%{name}
+
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{name}/bin
+install -p -D -m0755 src/bin/jowl.js %{buildroot}%{nodejs_sitelib}/%{name}/bin/jowl
+mkdir -p %{buildroot}%{_bindir}
+ln -sf %{nodejs_sitelib}/%{name}/bin/jowl %{buildroot}%{_bindir}/jowl
+
+# Fix the shebang because brp-mangle-shebangs fails to detect this properly (rhbz#1998924)
+# This is fixed in fc36 and above
+sed -e "s|^#!/usr/bin/env node$|#!/usr/bin/node|" \
+    -i %{buildroot}%{nodejs_sitelib}/%{name}/bin/jowl \
+    -i %{buildroot}%{nodejs_sitelib}/%{name}/jowl.js
+
+%check
+# Setup bundled dev node_modules for testing
+#   Note: this cannot be in %%prep or the dev node_modules
+#            can get pulled into the regular rpm
+tar xfz %{SOURCE2}
+# Ensure that this dir exists to be a target of the symlink
+mkdir -p node_modules_prod/.bin
+pushd node_modules
+ln -s -f ../node_modules_dev/* .
+popd
+pushd node_modules/.bin
+ln -s ../../node_modules_dev/.bin/* .
+popd
+# Run tests
+yarn run test
+
+
+%files
+%doc docs/reference.md
+%license LICENSE %{name}-%{version}-bundled-licenses.txt
+%{nodejs_sitelib}/%{name}
```

Comment:
I'd recommend adding a trailing `/` here, so that RPM unambiguously knows this is a folder.

(It's stupid that this is a thing you should do, blame POSIX)